tinker/templates/admin/redirects/home.html

```diff
@@ -72,6 +83,20 @@
 
         $(document).ready(function () {
 
+            $('#sftp-sync').click(function(){
+                var url = "{{ url_for('RedirectsView:manual_sftp_publish') }}";
+                $.post(url, null, function(data){
+                    $.notify({
+                        message: data
+                    }, {
+                        type: 'success',
```

Comment:
maybe a bit nitpicky, but it could be nice to have the type of notify be red, when it fails

Reply:
Done